Source/LinqToDB/Sql/Sql.QueryExtensionScope.cs

```diff
@@ -12,7 +12,8 @@ public enum QueryExtensionScope
 			IndexHint,
 			JoinHint,
 			SubQueryHint,
-			QueryHint
+			QueryHint,
+			TableNameHint
```

Comment:
```suggestion
			TableNameHint,
```

With a comma here, future changes only show the added line, rather than the modification of this line.